metaflow/datastore/common.py

```diff
@@ -0,0 +1,28 @@
+from ..exception import MetaflowException
```

Comment:
Might be worth splitting this into: `exceptions.py` and `inputs.py` since `common.py` feels too vague.

Reply:
Fair enough. I'll make the split.

Reply:
Done